mpsl/include/mpsl_fem_protocol_api.h

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file mpsl_fem_protocol_api.h
+ *
+ * @defgroup mpsl_fem MPSL Protocol interface for Power Amplifier (PA) and Low Noise Amplifier (LNA).
+ * @ingroup  mpsl
+ *
+ * This module enables control of peripherals before and after the radio transmission and the radio reception
```

Comment:
Instead of explaining this over again, I think it would be best to refer to the explanation in the README.rst instead, that explanation is the better one as it has diagrams in addition.